test/apiTests.js

```diff
@@ -0,0 +1,565 @@
+import chai from 'chai';
+import request from 'supertest';
+import 'babel-polyfill';
+
+import app from './../app';
+import database from './../server/src/database';
+import testData from './testData';
+
+const expect = chai.expect;
+
+describe('Given an API request', () => {
+  before(async () => {
+    await database.connection.sync({ force: true });
+  });
+
+  describe('When the request contains appropriate user registration data', () => {
+    it('Then the user is registered', async () => {
+      await request(app)
+        .post('/api/v1/users/signup')
+        .set('Accept', 'application/x-www-form-urlencoded')
+        .send(testData.newUser1)
+        .expect(201)
+        .then((response) => {
+          expect(response.body).to.be.an('object');
+          expect(response.body.message).to.equal('Registration successful');
+        });
+    });
+
+    it('Then the user is registered', async () => {
+      await request(app)
+        .post('/api/v1/users/signup')
+        .set('Accept', 'application/x-www-form-urlencoded')
+        .send(testData.newUser2)
+        .expect(201)
+        .then((response) => {
+          expect(response.body).to.be.an('object');
+          expect(response.body.message).to.equal('Registration successful');
+        });
+    });
+
+    it('Then the user is registered', async () => {
+      await request(app)
+        .post('/api/v1/users/signup')
+        .set('Accept', 'application/x-www-form-urlencoded')
+        .send(testData.newUser3)
+        .expect(201)
+        .then((response) => {
+          expect(response.body).to.be.an('object');
+          expect(response.body.message).to.equal('Registration successful');
```

Comment:
Having tested that the response body is an object and that the object contains a message property with value 'Registration successful'. It's important you test for the user object in the res body